app/actions/remote/channel.ts

```diff
@@ -850,13 +843,12 @@ export const createGroupChannel = async (serverUrl: string, userIds: string[]) =
                 if (categoryModels.models?.length) {
                     models.push(...categoryModels.models);
                 }
-                if (users?.length) {
+                if (users.length) {
```

Comment:
Seems like this check is not needed (handleUsers should be able to handle an empty list, and push too). Do we want to keep it?